Dolphin scripts/Entrance Randomizer/lib/graph_creation.py

```diff
@@ -11,23 +11,25 @@
 UPGRADE_AREAS_COLOR = "#0080ff"  # Blue
 IMPORTANT_STORY_TRIGGER_AREAS_COLOR = "#ff0000"  # Red
 UPGRADE_AREAS = {
-    LevelCRC.PLANE_COCKPIT,
+    LevelCRC.APU_ILLAPU_SHRINE,
```

Comment:
I don't think it's a good idea to sort UPGRADE_AREAS alphabetically.
Because unlike the full level list, this one can actually be read and discussed by people, and if I were to make sure I list all areas with upgrades I'd do so in some logical order to make sure I didn't forget any. Alphabetical is not such an order.

As for sorting the IMPORTANT_STORY_TRIGGER_AREAS alphabetically, the exact same thing, except those are currently only 4 so you'd be able to read all of them at a single glance anyway. So for that reason I don't really care how you sort those 4.

Reply:
I've noted the expected upgrade in comment to make it obvious what this "human ordering" is.